templates/tomcat-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: xwiki-tomcat-conf
```

Comment:
Hardcoded name

Reply:
Leaving as it is, following the original standard. This PR doesn't concern with drastic changes to the original standards